account_analytic_parent/models/account_analytic_account.py

```diff
@@ -11,10 +11,25 @@
 
 class AccountAnalyticAccount(models.Model):
     _inherit = "account.analytic.account"
+    _parent_name = "parent_id"
+    _parent_store = True
+    _parent_order = 'name'
+    _rec_name = 'display_name'
+    _order = 'parent_left'
 
+    parent_left = fields.Integer(
+        string='Left Parent',
+        index=1,
```

Comment:
Why `index=1` instead of  `index=True`? It shouldn't be a problem, but better to be homogeneous